### DISCUSSION TITLE
How can I build under FreeBSD?

### DISCUSSION THREAD

**Discussion by iron-udjin** [dragonflydb/dragonfly#974] [Q&A]
1. Is that possible to build Dragonfly under FreeBSD?
2. Are you planning to add support listening UNIX Socket?

We use Redis under high load use Unix Socket. And it works significantly faster (on our workload) than via TCP socket. Also, sometimes app run in isolated environment without network and communicate with all external databases via UNIX Socket (redis, MySQL, PostgreSQL, etc...). One more reason that TCP communication has some delay compared with UNIX Socket and it is noticeable at high load.

**Comment by romange** [comment]
1. Dragonfly is linux only
2. Dragonfly supports unix sockets with `--unixsocket` option, similarly to redis.

**Reply by iron-udjin** [reply]
Ok, thanks.
